ext/sqlite3/sqlite3.stub.php

```diff
@@ -312,7 +312,7 @@ public function __construct(string $filename, int $flags = SQLITE3_OPEN_READWRIT
     public function open(string $filename, int $flags = SQLITE3_OPEN_READWRITE | SQLITE3_OPEN_CREATE, string $encryptionKey = ""): void {}
 
     /** @return bool */
```

Comment:
This comment needs to be removed and I think also replaced with ``/** @tentative-return-type */``

Is this correct @kocsismate ?

Reply:
looking at another void function, you seem to be correct.

Reply:
Yeah, you are right